app/model/NotificationTestType.scala

```diff
@@ -44,6 +46,12 @@ object Phase3FailedTestType
   extends FailedTestType(PHASE3_TESTS_FAILED, PHASE3_TESTS_FAILED_NOTIFIED, PHASE3_TESTS_RESULTS_RECEIVED,
       "fset_faststream_app_online_phase3_test_failed")
 
+object Phase1SuccessTestType
+  extends SuccessTestType(ApplicationStatus.PHASE1_TESTS_PASSED, PHASE1_TESTS_SUCCESS_NOTIFIED,
+    READY_FOR_EXPORT, "fset_faststream_app_online_phase1_test_success") {
+  override val applicationRoutes = List(ApplicationRoute.Edip)
+}
```

Comment:
For other notification types we are setting the application routes to empty means the application route filter is not considered.